src/components/KeyboardShortcuts/index.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+import styles from './styles.module.scss';
+import { gettext } from '../../utils';
+
+const keys = ['k', 'j', 'e', 'h'];
+
+type Props = {};
+
+export class KeyboardShortcutsBase extends React.Component<Props> {
+  keydownListener = (event: KeyboardEvent) => {
+    if (keys.includes(event.key)) {
```

Comment:
You'll probably also need to check that a meta key (ctrl, command) isn't present